src/genai_linkedin/model.py

```diff
@@ -0,0 +1,48 @@
+import os
+from typing import Union
```

Comment:
rather `import typing`
and then below in your code `typing.Union`

This makes it easy to know where the method `Union` comes from when you are reading the code.